lib/recipe/presenter.rb

```diff
@@ -6,47 +6,38 @@ def initialize(recipe:)
       @recipe = recipe
     end
 
-    def present
-      output = <<~STR
+    def output
+      <<~STR
         #{recipe_info}
         #{ingredients}
         #{instructions}
       STR
-
-      print output
     end
 
     private
 
-    def recipe_info
-      <<~STR
-        Title: #{recipe["title"]}
-        Serves: #{recipe["serves"]}
-        Total time: #{recipe["total_time"]}
-        Source: https://grubdaily.com
-
-        #{recipe["summary"]}
-      STR
+    def ingredient_entries_array
```

Comment:
This should maybe live on a Recipe class, representing a database record (ie: one object from the JSON file)